.gitpod.yml

```diff
@@ -1,4 +1,4 @@
-image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:wth-update-terraform.0
+image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:gpl-test-sh.1
```

Comment:
ideally this would be one (squashed) commit with the actual change. There's no need to litter the commit history with "Updated image".